libmscore/stafftype.cpp

```diff
@@ -1413,65 +1414,65 @@ void StaffType::initStaffTypes()
 
     // keep in sync with enum class StaffTypes
     _presets = {
-//                       group,              xml-name,  human-readable-name,          lin stpOff  dist clef   bars stmless time  key    ledger
-        StaffType(StaffGroup::STANDARD,   "stdNormal", QObject::tr(
+//                                      group,              xml-name,  human-readable-name,          lin stpOff  dist clef   bars stmless time  key    ledger
+        StaffType(StaffTypes::STANDARD, StaffGroup::STANDARD,   "stdNormal", QObject::tr(
                       "Standard"),        5, 0,     1,   true,  true, false, true, true,  true),
 //       StaffType(StaffGroup::PERCUSSION, "perc1Line", QObject::tr("Perc. 1 line"),    1, -4,    1,   true,  true, false, true, false, true),
-        StaffType(StaffGroup::PERCUSSION, "perc1Line", QObject::tr(
+        StaffType(StaffTypes::PERC_1LINE, StaffGroup::PERCUSSION, "perc1Line", QObject::tr(
                       "Perc. 1 line"),    1, 0,     1,   true,  true, false, true, false, true),
-        StaffType(StaffGroup::PERCUSSION, "perc3Line", QObject::tr(
+        StaffType(StaffTypes::PERC_3LINE, StaffGroup::PERCUSSION, "perc3Line", QObject::tr(
                       "Perc. 3 lines"),   3, 0,     2,   true,  true, false, true, false, true),
-        StaffType(StaffGroup::PERCUSSION, "perc5Line", QObject::tr(
+        StaffType(StaffTypes::PERC_5LINE, StaffGroup::PERCUSSION, "perc5Line", QObject::tr(
                       "Perc. 5 lines"),   5, 0,     1,   true,  true, false, true, false, true),
 //                 group            xml-name,     human-readable-name                  lin stpOff dist clef   bars stemless time      duration font     size off genDur     fret font          size off  duration symbol repeat      thru       minim style              onLin  rests  stmDn  stmThr upsDn  sTFing nums  bkTied
 //       StaffType(StaffGroup::TAB, "tab6StrSimple", QObject::tr("Tab. 6-str. simple"), 6, 2,     1.5, true,  true, true,  false, "MuseScore Tab Modern", 15, 0, false, "MuseScore Tab Sans",    9, 0, TablatureSymbolRepeat::NEVER, false, TablatureMinimStyle::NONE,   true,  false, true,  false, false, false, true, false),
 //       StaffType(StaffGroup::TAB, "tab6StrCommon", QObject::tr("Tab. 6-str. common"), 6, 2,     1.5, true,  true, false, false, "MuseScore Tab Modern", 15, 0, false, "MuseScore Tab Serif",   9, 0, TablatureSymbolRepeat::NEVER, false, TablatureMinimStyle::SHORTER,true,  false, true,  false, false, false, true, true),
 //       StaffType(StaffGroup::TAB, "tab6StrFull",   QObject::tr("Tab. 6-str. full"),   6, 2,     1.5, true,  true, false, true,  "MuseScore Tab Modern", 15, 0, false, "MuseScore Tab Serif",   9, 0, TablatureSymbolRepeat::NEVER, false, TablatureMinimStyle::SLASHED,true,  true,  true,  true,  false, false, true, true),
-        StaffType(StaffGroup::TAB, "tab6StrSimple", QObject::tr(
+        StaffType(StaffTypes::TAB_6SIMPLE, StaffGroup::TAB, "tab6StrSimple", QObject::tr(
```

Comment:
This change could break things that use `StaffTypes::PERCUSSION` and `StaffTypes::TAB`, for example when judging whether a tremolo is attached to a chord on a TAB staff. Please make sure to take care of these.

Reply:
This arg is not used in any existed code. It is needed for displaying a current staff type in a staff settings popup

Reply:
Ah I see, I confused `StaffGroup` and `StaffTypes` :)